src/reducers/versions.tsx

```diff
@@ -785,6 +786,18 @@ export const createInternalChangeInfo = (
   };
 };
 
+export const createInternalHunk = (hunk: ExternalHunk): HunkInfo => {
```

Comment:
Do you think we should have a test for this new function? I don't see one added.

Reply:
Good catch. It actually did have tests but I meant to move them from out of the `createInternalDiff` suite. I will move them.